clients/privacy-center/config/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Take control of your data",
+  "description": "When you use our services, you’re trusting us with your information. We understand this is a big responsibility and work hard to protect your information and put you in control.",
+  "fidesops_host_development": "http://localhost:8080/api/v1",
```

Comment:
One last thing @elliotbonneville — I think we wanted to add the auth URL here?

Reply:
![Screenshot 2022-03-14 at 17 16 58](https://user-images.githubusercontent.com/1667340/158262768-54da4c51-e0eb-4ad0-a31a-581c5951840f.png)

Specifically wrt this change